lib/index.js

```diff
@@ -22,6 +22,9 @@ function register (server, options, next) {
         if (err) {
           return cb(err, null);
         }
+        if (rows.length === 0) { // i.e. no orgs -OR- filter --> wrong tagId or no orgs data
+          return cb(err, null);
```

Comment:
Im not sure what is best to return in this case. I thought no orgs would return the object still as we want to return the filter tag.

Doesn't it mean the tag Id was wrong? In which case what is best to return? is it an error?